modules/claims_api/lib/claims_api/v1/poa_pdf_constructor/base.rb

```diff
@@ -106,13 +106,20 @@ def stringify_address(address)
         def combine_pdf(id, page1_path, page2_path, page3_path, page4_path)
           output_path = "/tmp/#{id}_final.pdf"
 
+          # Temporarily suppress CombinePDF warning in CI
+          # Couldn't connect reference for ...
```

Comment:
Have we determined the root cause here?

Reply:
It sounds innocuous. Should we just ignore it everywhere?

Reply:
Hello, I'm not on y'all's team but I've also been researching the cause of these messages. This is the most pertinent conversation/documentation I've been able to find on the topic. From what they can tell, the cause is most likely different PDF authoring tools assigning NULL objects. The warnings happen in case this in unintentional. 

https://github.com/boazsegev/combine_pdf/issues/15

Here is the exact line that appears to be setting the warning:

https://github.com/boazsegev/combine_pdf/blob/master/lib/combine_pdf/parser.rb#L664

Reply:
Yes, there's NULL objects. The raw data may not contain some information such as an apartment number, so when it's transformed to "page data" there's nil values:

 `:"F[0].Page_1[0].Claimants_MailingAddress_ApartmentOrUnitNumber[1]"=>nil,`

Also by temporarily I meant the suppression is limited to the `#combine_pdf` method. Poor wording on my part.